services/zapier.rb

```diff
@@ -0,0 +1,65 @@
+class Service::Zapier < Service
+  def receive_validate(errors = {})
+    if settings[:url].to_s.empty?
+      errors[:url] = "Is required"
+      return false
+    end
+
+    uri = URI.parse(settings[:url]) rescue nil
+    unless uri
+      errors[:url] = "Is not a valid URL"
+      return false
+    end
+
+    return true
+  end
+
+  def receive_alert
+    raise_config_error unless receive_validate({})
+    send_alert(body)
```

Comment:
We should probably continue to support both alert types until we know v1 is officially deprecated. See https://github.com/librato/librato-services/blob/master/services/sns.rb#L43-L59 for an example.